core/utils/poly-look/src/css/themes.css

```diff
@@ -0,0 +1,21 @@
+:root,
+.poly-theme {
+  --theme-font-base: var(--jost);
+  --theme-font-medium: var(--jost-medium);
+  color: var(--theme-secondary-color);
+  font-family: var(--jost);
+  font-weight: 400;
+  line-height: 120%;
+  letter-spacing: -0.01em;
+}
+
+:root,
+.poly-theme.light {
```

Comment:
I don't like too much how this will lead to classnames like `"poly-theme light"`